README.md

```diff
@@ -27,7 +27,53 @@ This project welcomes contributions and suggestions.  Most contributions require
 ## Script support
 PSF allows one PowerShell script to be run before an exe runs, and one PowerShell script to be ran after the exe runs.
 
-Each exe defined in the application manifest can have their own scripts.
+Each exe defined in the application manifest can have their own scripts.
+
+### Stopping application on a script error.
+PSF gives users the options to stop the application if the starting script encounters an error.  To enable PSF to stop if the starting script encounters an error
+add the pair stopOnScriptError and set it to true.  Example `"stopOnScriptError": true`  The default PSF value for stopOnScriptError is false.  This pair can be specified on a per exe basis.
```

Comment:
To disable further PSF script runs on error, add the option "stopOnScriptError":true.